src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="wrapper">
+      <form
+        class="searchBarWrapper"
+        data-qa="big"
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="post"
+      >
+        <input
```

Comment:
Nice job!
But have some comments about code style and semantic.

Semantically right to do layout where every input has own label.